python/cudf_polars/cudf_polars/containers/column.py

```diff
@@ -113,9 +114,12 @@ def deserialize_ctor_kwargs(
         column_kwargs: ColumnOptions,
     ) -> DeserializedColumnOptions:
         """Deserialize the constructor kwargs for a Column."""
-        dtype = DataType(  # pragma: no cover
-            _dtype_short_repr_to_dtype(column_kwargs["dtype"])
-        )
+        dtype_kwarg = column_kwargs["dtype"]
+        if isinstance(dtype_kwarg, dict):
+            dtype = DataType(pl.Decimal(dtype_kwarg["precision"], dtype_kwarg["scale"]))
+        else:
+            dtype = DataType(_dtype_short_repr_to_dtype(dtype_kwarg))
```

Comment:
We lost the `no cover` is that deliberate?

Reply:
Ah we still need it because this would only get hit for the `--cluster distributed` run